catalog/ui/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -122,6 +125,12 @@ define([
             'AFTER': temporalClass,
             'DURING': temporalClass
         },
+
+        // TODO: for extra credit, these could be figured out while building the syntax tree
```

Comment:
I'm just asking because we generally want a `TODO` associated with a ticket.  We can make a ticket for it on the DDF JIRA if we plan on addressing it in the future